tests/operations.go

```diff
@@ -178,10 +178,14 @@ func (ops *Operations) GetPodRunningCount(namespace, lselector string) int {
 
 // IsPVCBound checks if the pvc is bound or not
 func (ops *Operations) IsPVCBound(pvcName string) bool {
-	volume, err := ops.PVCClient.
-		Get(pvcName, metav1.GetOptions{})
-	Expect(err).ShouldNot(HaveOccurred())
-	return pvc.NewForAPIObject(volume).IsBound()
+	return Eventually(func() bool {
```

Comment:
actual logic & eventually logic should be separate.
There can be two methods e.g. `IsPVCBound` & `IsPVCBoundEventually`

Reply:
done

Reply:
I could not find the rework

Reply:
this is new method which is introduced. there is no change in older actual logic.